src/actions/index.js

```diff
@@ -1,11 +1,30 @@
 import agileSDK from 'agile-sdk';
-const agile = agileSDK('/api');
+
+
+var agile = agileSDK({
+  api: 'http://agile-core:8080',
+  idm: 'http://agile-security:3000'
+});
+
+//This sets the token for the calls to the sdk and reloads the SDK object
+
+export const setToken = (new_token) => {
+  var token = new_token;
```

Comment:
Once https://github.com/Agile-IoT/agile-sdk/pull/30 is set we can just use `agile.setToken`